.github/workflows/build.yml

```diff
@@ -18,9 +18,9 @@ jobs:
       fail-fast: false
       matrix:
         os: [
-          # Migrate back to macOS-latest once 'latest' workflow label is updated (April – June 2024)
-          # https://github.blog/changelog/2024-01-30-github-actions-macos-14-sonoma-is-now-available/
-          macOS-14,
+          # Check https://github.com/actions/runner-images?tab=readme-ov-file#available-images for actual versions for latest
+          macOS-latest, # arm64
+          macOS-14-latest-large, # intel
```

Comment:
```suggestion
```

Afaik, Intel Macs are no longer supported?  If correct, it'd be best not increasing the test matrix and build times.